hw/ip/pinmux/data/pinmux.hjson.tpl

```diff
@@ -566,7 +566,7 @@
                   regwen:       "MIO_PAD_ATTR_REGWEN",
                   regwen_multi: "true",
                   cname:        "MIO_PAD",
-                  resval:       0
+                  resval:       "0x80",
```

Comment:
If the intention is to not introduce software changes, the logic might've been a bit simpler if the pad attribute were `input_disable` (i.e. with logic levels that map a 1 to disable).

`input_en` is totally fine too, though. Just figured I'd present that option in case you hadn't considered it.

Reply:
True; that's probably less surprising from a SW/programming perspective

Reply:
Fixed; it's now `input_disable`